doc/5.reference/line~-help.pd

```diff
@@ -38,12 +38,13 @@
 #X text 126 178 signal - ramp values.;
 #X text 141 212 NONE;
 #X restore 333 13 pd reference;
-#X obj 8 554 cnv 1 500 1 empty empty empty 8 12 0 13 #000000 #000000 0;
-#X msg 106 329 0 \, 1 1000;
-#X text 185 322 setting start \, target and duration with two messages., f 28;
-#X text 28 57 The [line~] object generates linear ramps whose levels and timing are determined by messages you send it. A list of two floats distributes the value over the inlets \, as usual in Pd. Note that the right inlet (that sets the ramp time in milliseconds) does not remember old values (unlike every other inlet in Pd). Thus \, if you don't priorly specify a time in the right inlet and sent [line~] a float \, it jumps immediately to the target value., f 65;
-#X obj 281 427 set-dsp-tgl;
-#X text 312 431 DSP on/off;
+#X obj 8 636 cnv 1 500 1 empty empty empty 8 12 0 13 #000000 #000000 0;
+#X msg 106 351 0 \, 1 1000;
+#X text 185 344 setting start \, target and duration with two messages., f 28;
+#X obj 281 449 set-dsp-tgl;
+#X text 312 453 DSP on/off;
+#X text 28 57 The [line~] object generates linear ramps whose levels and timing are determined by messages you send it. Note that the right inlet value (for ramp time in milliseconds) is cleared when the object receives a value in the left inlet (unlike as other objects in Pd). Thus \, if you don't priorly specify a time in the right inlet again before sending [line~] a float \, it jumps immediately to the target value. You can also use a list of two floats to distribute the values over the inlets (as usual in Pd) \, which keeps resetting the time ramp., f 66;
+#X text 37 568 Note that the [line~] object cannot start and stop ramps in the middle of an audio block. For that \, you can use [vline~] \, which generates ramps with more precision and also has a delay parameter to schedule different line ramps., f 61;
```

Comment:
what does *more precision* mean in this context?
it could be "more precise along the time axis", or "more precise along the value axis"; since the time axis is also handled by the prior 1½ sentences, i *think* the reader is let to believe that this is about "value precision".

i would just shorten this sentence to:
> For that, you can use [vline~], also has a delay parameter to schedule different line ramps.